javascript-local-storage/data.js

```diff
@@ -0,0 +1,11 @@
+/* exported todos */
+var todos = [];
+
+var previousTodosJson = localStorage.getItem('javascript-local-storage');
+
+todos = JSON.parse(previousTodosJson);
```

Comment:
What if there's nothing there? 🤔 
![Screen Shot 2020-10-23 at 3 37 11 PM](https://user-images.githubusercontent.com/32880187/97059572-a0d25e00-1545-11eb-9c8f-80e1596af439.png)

Reply:
Yes I see now. I added the if statement. If the to do list didn't have any data to be saved, json.parse shouldn't occur because if it did, there would be an error. 

The error in the console = 'Uncaught TypeError: Cannot read property 'length' of null'